orc8r/cloud/docker/docker-compose.thanos.yml

```diff
@@ -0,0 +1,83 @@
+# This compose file installs the required thanos components for orchestrator, as well as overriding
+# some configurations to make thanos work properly. It is NOT recommended to use this in any
+# production setting, but rather just as a way of development testing and experimentation.
+
+version: "3.7"
+
+services:
+  prometheus:
```

Comment:
it's not a problem that there's already a `prometheus` service in docker-compose.metrics.yml?

Reply:
This just overrides the values when you run it like `docker-compose -f docker-compose.metrics.yml docker-compose.thanos.yml`. Only one service, uses the later values.